pkg/scheduler/partition_test.go

```diff
@@ -1703,3 +1704,34 @@ func TestRemoveAllocationAsk(t *testing.T) {
 	partition.removeAllocationAsk(release)
 	assert.Assert(t, resources.IsZero(app.GetPendingResource()), "app should not have pending asks")
 }
+
+func TestUpdateNodeSortingPolicy(t *testing.T) {
+	partition, err := newBasePartition()
+	assert.NilError(t, err, "partition create failed")
+
+	assert.Equal(t, partition.nodeSortingPolicy.PolicyType(), policies.FairnessNodeSortingPolicy)
```

Comment:
I rename the `SortingPolicy` to `NodeSortingPolicy` since there is already a file called `sorting_policy`. The naming is weird and can confuse readers. For example, I referenced to incorrect constant when writing this test since it is hard to get the point from old naming.

However, I'm ok to revert this renaming as it is not related to this PR.